src/rebuild/srv.c

```diff
@@ -1224,7 +1299,7 @@ rebuild_task_ult(void *arg)
 	}
 
 	/* Wait until rebuild finished */
-	rebuild_leader_status_check(pool, task->dst_map_ver,
+	rebuild_leader_status_check(pool, task->dst_map_ver, task->dst_map_ver,
```

Comment:
3rd parameter is a typo of "task->dst_max_ver"?

Reply:
ah, right.